docs/troubleshooting/index.mdx

```diff
@@ -35,6 +37,48 @@ Alpine is missing some dependencies that [are required](https://github.com/datre
 Adding the following command to the CI should fix this: `apk add curl bash`
 This will add curl and bash commands which our install scripts require in order to run.
 
+---
+
+## Schema validation failures
+
+If you get the following message in your CLI: 
+```terminal
+❌  k8s schema validation error: could not find schema for [KIND]  
+``` 
+It means that the resource's schema which is used to perform the schema valiation was not found, and it can be caused by any of the following:
+
+### Wrong K8s version is used in the schema validation check
+To perform the schema validation check, the CLI pulls the relevant schema according to your default K8s schema, or the value of the `--schema-version` flag.  
+To fix this failure, update your default K8s version in the dashboard to the same K8s version as your cluster, or pass the `--schema-version` flag.  
+Instructions for how to update your default K8s version can be found [here](https://hub.datree.io/setup/schema-validation#changing-the-default-schema-version)
+
+### Deprecated API versions
+K8s releases may contain deprecations of APIs of different K8s objects- for example, the beta API versions of Ingress are no longer served from version 1.22.
+<Tabs>
+  <TabItem value='Deprecated API version' label='Deprecated API version' default>   
+
+    apiVersion: networking.k8s.io/v1beta1 # deprecated API version
+    kind: Ingress  
+    metadata:  
+      name: example  
+  </TabItem>
+  <TabItem value='Valid API version' label='Valid API version'>
+
+    apiVersion: networking.k8s.io/v1 # supported API version
+    kind: Ingress
+    metadata:
+      name: example
+  </TabItem>
+</Tabs>  
+
+When validating objects with deprecated API configured, the CLI will not be able to pull the schema since it's removed.  
```

Comment:
When validating objects with a deprecated API version, the CLI will not be able to pull the schema.